index.html

```diff
@@ -0,0 +1,368 @@
+
+![](images/multi-monitor-setup.png)
+*<br>An example [=multi-monitor setup=] optimized for software development tasks with a [=traditional laptop=] connected to two [=monitors=] with one monitor in landscape orientation displaying code review side by side and one monitor in portrait orientation displaying a long change log listing. ([Source](TODO))*
+
+### Monitor
+
+A <dfn>monitor</dfn> is an output device that displays information in pictorial form. A monitor usually comprises the display, circuitry, casing, and power supply. ([Wikipedia](https://en.wikipedia.org/wiki/Computer_monitor))
+
+### Display
+
+A <dfn>display</dfn>, informally a [=screen=], is a display device for presentation of images, text, or video transmitted electronically, without producing a permanent record. Electronic visual displays include television sets, computer monitors, and digital signage. ([Wikipedia](https://en.wikipedia.org/wiki/Electronic_visual_display))
+
+### Screen
+
+<dfn>Screen</dfn> is a synonym to [=display=]. [=Screen=] is used more in software context (logical) while display is primarily used in hardware context (physical). See also the [=Screen interface=].
+
+### Multiple internal display
+
+A <dfn>multiple internal display</dfn> (MID) device is a more technical synonym to a [=dual-screen device=].
+
+### Single foldable display
+
+A <dfn>single foldable display</dfn> (SFD) refers to a [=foldable-screen device=] with a single screen.
+
+### Posture
+
+A <dfn>posture</dfn> is defined as the primary way in which the [=foldable device=] is positioned and presented to the user. Because [=foldable devices=] have a hinge they can present different [=postures=] depending on the hinge angle, device orientation, and the way on how the device is being held by the user. Example postures include “book”, “flat”, “laptop”, and “tent”. The list of postures is expected to evolve over time, and not all foldable devices are able to support all possible postures.
+
+![](images/postures.png)
+*<br>Example postures from left to right: “closed”, “laptop”, “table top”, “flat”, “canvas”, and “book”. ([Source](TODO))*
```

Comment:
We should probably use the same nomenclature... like screen fold spec uses tablet for canvas, but maybe canvas is better... I have also seen "compose" being used instead of laptop (which doesn't make sense for a phone)

Reply:
Can you share a concrete proposal how to word this? Or should this be an issue instead?

Reply:
+1 here. We've discussed the need to rethink the name of the postures to make them as abstract as possible. The fact that 'laptop' is both an actual device and a 'posture' mode can be confusing and misleading. Can a laptop have another posture that's not laptop? Does a mobile device get into laptop posture? Similar case with tablet as a device type and posture. Happy to help Kenneth abstract this nomenclature to broaden it.

Reply:
I made @diekus's comment an issue in the spec in the section that defines posture to make sure this does not go unnoticed.

(Note: images/postures.png was removed in 3229fe0eda3c595795057ca5c1267e8962f03c71)